src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -157,38 +159,34 @@ export const pageContent = {
         >
           More on the merge
         </Link>
+        <Text size="medium" className="my10">
+          With transfers disabled for now, you won't be able to voluntarily exit
+          and then restart later. This means you need to be in it for the long
+          haul.
+        </Text>
       </>
     ),
-    acknowledgementText: `I understand that I CANNOT TRANSFER my staked ${TICKER_NAME} for a while, and I CANNOT WITHDRAW until the merge.`,
-  },
-  [AcknowledgementIdsEnum.commitment]: {
-    title: 'Validating is a long-term commitment',
-    content: (
-      <Text size="large" className="my10">
-        With transfers disabled for now, you won't be able to voluntarily exit
-        and then restart later. This means you need to be in it for the long
-        haul.
-      </Text>
-    ),
     acknowledgementText:
-      'I understand that if I exit, I will not be able to rejoin until much later. This is a long term commitment.',
+      'I understand that I CANNOT TRANSFER my stake for a while, and I CANNOT WITHDRAW until the merge. I understand that if I exit, I will not be able to rejoin until much later. This is a long term commitment.',
```

Comment:
I'm not sure we should capitalize text like this. Not sure if it potentially creates confusion for translations. I'd lean towards assuming people will read this without the need for all caps.